docker/multiversx/Dockerfile

```diff
@@ -1,12 +1,12 @@
-FROM golang:1.17.6 as builder
+FROM golang:1.20.7 as builder
 
 RUN apt-get update && apt-get install -y
 WORKDIR /go/mx-chain-go
 COPY . .
 RUN GO111MODULE=on go mod tidy
```

Comment:
The docker build won't work. There are these choices:
- update `go` version in `go.mod` to `1.20` (pending other satellite projects)
- update this line to: `RUN GO111MODULE=on go mod tidy -compat=1.17`
- completely remove this line (not recommended, since we want to test `go mod tidy` as well)

Docker build can be tested by running this command in the project's root:
`docker build . --file docker/multiversx/Dockerfile`